databases/week1/hyf_lesson1.sql

```diff
@@ -0,0 +1,10 @@
+SELECT COUNT(title) FROM task;
+SELECT*FROM task WHERE due_date IS NULL;
+SELECT task.id, task.title, status.name  FROM task, status WHERE status.name LIKE "%Done%";
+SELECT task.id, task.title, status.name FROM task , status WHERE status.name NOT LIKE "%Done%";
+SELECT task.id, task.title, task.created FROM task ORDER BY created DESC;
+SELECT task.id, task.title, task.created FROM task ORDER BY created DESC LIMIT 1;
+SELECT task.title,task.due_date,task.description from task WHERE task.title LIKE "%database%" OR task.description LIKE "%database%";
+SELECT task.title , status.name FROM task,status WHERE task.status_id=status_id;
```

Comment:
Line 8. Task: _Get the title and status (as text) of all tasks_
This query brings backs each task 3 times with different statuses in each case. 
Try to rethink the approach 🙏